src/screens/auth/UpdatePassword.tsx

```
@@ -0,0 +1,147 @@
+    const navigate = useNavigate();
+    const toast = useToast();
+
+    const [oldPassword, setOldPassword] = useState('');
+    const [newPassword, setNewPassword] = useState('');
+    const [confirmPassword, setConfirmPassword] = useState('');
+    const [loading, setLoading] = useState(false);
+
+    const username = localStorage.getItem('pendingUser');
+
+    const handleUpdatePassword = async () => {
+        if (!oldPassword.trim() || !newPassword.trim() || !confirmPassword.trim()) {
```

ESLint

[error] Replace `········if·(!oldPassword.trim()·||·!newPassword.trim()·||·!confirmPassword.trim()` with `↹↹if·(⏎↹↹↹!oldPassword.trim()·||⏎↹↹↹!newPassword.trim()·||⏎↹↹↹!confirmPassword.trim()⏎↹↹`

(prettier/prettier)